cloudinary/utils.py

```diff
@@ -1239,6 +1239,7 @@ def process_layer(layer, layer_parameter):
     "pageY": "py",
     "tags": "tags",
     "width": "w",
+    "preview:duration": "preview:duration",
```

Comment:
@YomesInc, please avoid using hardcoded workaround fixes, it does not really solve the problem.
The fix should be implemented in `normalize_expression` function.